src/utils/files/uploadSchemaFile.ts

```diff
@@ -0,0 +1,73 @@
+import fs from 'fs';
+import config from 'config';
+import i18next from 'i18next';
+import { logger } from '@services/logger.service';
+import { BlobServiceClient } from '@azure/storage-blob';
+import { printSchema } from 'graphql';
+import buildSchema from '@utils/schema/buildSchema';
+
+/** Storage connection string */
+const STORAGE_CONNECTION_STRING: string = config.get('public.fileName');
```

Comment:
@estelafs 

for this one also, the value should be exactly:

const AZURE_STORAGE_CONNECTION_STRING: string = config.get(
  'blobStorage.connectionString'
);

this is where the error comes from

to upload the file, you need the connection string

the public.fileName is only there to know where the file is